rfc/rfc-0009-supply-chain-switches-template-on-flag.md

```diff
@@ -1,3 +1,182 @@
-[Draft PR](https://github.com/vmware-tanzu/cartographer/pull/75)
+# Draft RFC 0009 Supply Chain Switches Template in single step
 
-[Draft Text](https://github.com/vmware-tanzu/cartographer/blob/rfc-0009-supply-chain-switches-template-on-flag/rfc/rfc-0009-supply-chain-switches-template-on-flag.md)
+## Summary
+
+Platform creators will need to handle multiple types of workloads. While Cartographer
+supports that behavior currently through allowing multiple supply chains, it should
+reduce the number of supply chains necessary by allowing individual supply chains
+to choose between templates within a step.
+
+## Motivation
+
+The Developer Productivity program auto-generates workloads from CLI commands.
+Those workloads match with supplychains and templates that are written by the
+dev-prod team as well. The team would like to limit the number of supplychains
+that must be written and maintained. Small differences in workload can lead to
+needing an entirely different supply chain. For example, what code source is used.
+
+Further info on their use case can be read [here](https://docs.google.com/document/d/1TVqlNqTyCMlp_yNs9_F80QAIQSC5bxm05vSyWyeGMSw/edit).
+See also this [discussion](https://vmware.slack.com/archives/C01UX69LJCB/p1629408443051200).
```

Comment:
The doc and slack thread listed here are not publicly viewable.

Reply:
Removed links to them. Good catch.